HW03/Arthur_Bohan/notification.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# notification
+class Notification
+  attr_reader :notification_data
+
+  def initialize(_notification_data)
+    @notification_data
+  end
```

Comment:
mb do you mean smth like this?
```suggestion
  def initialize(notification_data)
    @notification_data = notification_data
  end
```